helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -412,6 +429,30 @@ private void refreshBaseline(ResourceControllerDataProvider clusterData,
     return newAssignment;
   }
 
+  private void measureBaselineDivergenceGauge(Map<String, ResourceAssignment> baseline,
```

Comment:
This method might potentially add some latency to BestPossibleCalcStage. Should we consider making this asynchronous? @jiajunwang

Reply:
Considering the data size, we will use async call to measure the metric.